### DISCUSSION TITLE
Suggestion: reorder params for border

### DISCUSSION THREAD

**Discussion by aileksandar** [master-co/css#255] [Ideas]
I've been using master css extensively, I've even made my own variation for React Native because the syntax and convention makes a lot of sense. 
The thing that makes more sense to me is to have a different ordering when it comes to border.

Currently is `border:size|style|color` so you use it like `border:1|solid|gray`, but I realized it makes a lot of clutter because:
style: solid is 99% of the time
size: 1px is 50% of the time
color: I update the color 100% of the time

So it makes more sense for me to be `border:color|size|style` since the size and style would very often be default and make my code from `border:1|solid|gray` to `border:gray`

I think v2 is a good time to introduce this kind of breaking change.

Thoughts?

**Comment by 1aron** [comment]
Agreed, but `border:gray` is an ambiguous shorthand for `border-color`.

You can predefine the following `master.css.js`:
```js
export default {
    classes: {
        border: 'border:1 border:solid'
    }
}
```
Apply it:
```html
<div class="border border:gray">
```

**Reply by aileksandar** [reply]
Thanks a lot for the reply.

I understand that you are following the convention and ordering from the css so I would not make sense to change ordering. 
But my question is now will there be possible via some config to override the behaviour? 

```js
export default {
    classes: {
        border: 'border:1 border:solid'
    }
}
```

This just creates another class but like I said, I would still like to somehow use it like `border:color|size|style`. I guess you would have to allow to somehow extend the framework with custom methods?

**Comment by neneodonkor** [comment]
How did you get it to work with RN?
